Speech_Generator.py

```diff
@@ -38,6 +46,44 @@ def __init__(self, credentials:str="auth.json",
         if (not(os.path.isdir(self.data_dir_path))):
             os.makedirs(self.data_dir_path)
 
+        self.file_mapping = {
+            "<<GENERICS>>":[]
+        }
+
+    def parse_rows(self, paths:list) -> dict:

```

Comment:
I may be missing context here, but does the return value need to be a dictionary if the value for each entry is always going to be "0"?

Reply:
Maybe the keys for this dictionary that this function returns are of some use later?

Reply:
Just changed it back to a set.